### PR TITLE
Updates to OOB InvitationMessage & Endpoint example - Consistent with RFC0434

### LINKED ISSUE #1138: OOB InvitationReceiveRequest model
The model specifies `service_dids` and `services`, but the example value in the Open API definition only specifies `service_dids`.
I guess `service_dids` is deprecated. I can't find anything about it in the OOB RFC and ACA-Py will return an error when `services` is not specified. `Services` should either accept a (fully-qualified) DID or a DIDcomm service object.

### PR DESCRIPTION
**Pull request by shaangill025**
- resolve #1138

### CODE REVIEW

File: aries_cloudagent/protocols/out_of_band/v1_0/messages/invitation.py [hunk 1]
```diff
@@ -86,6 +85,26 @@ def aka(self) -> int:
...
+    def _serialize(self, value, attr, obj, **kwargs):
+        if isinstance(value, Service):
+            return value.serialize()
+        return value
+
+    def _deserialize(self, value, attr, data, **kwargs):
+        if isinstance(value, dict):
+            return Service.deserialize(value)
+        elif isinstance(value, str):
+            if bool(IndyDID.PATTERN.match(value)):
```
Comment (andrewwhitehead) [inline aries_cloudagent/protocols/out_of_band/v1_0/messages/invitation.py:100]: I don't think this is the right place to be restricting the implementation to only Indy DIDs. Even now we should be able to accept an invitation with multiple DID services as long as one of them is supported.

File: open-api/openapi.json [hunk 2]
```diff
@@ -6241,22 +6241,29 @@
             "$ref" : "#/definitions/AttachDecorator"
           }
         },
-        "service_blocks" : {
+        "services" : {
```
Comment (andrewwhitehead) [inline open-api/openapi.json:6244]: This one trailing space seems slightly odd to me

**Comment by andrewwhitehead** [comment]
Looks good but maybe we can be less restrictive about what DID formats are supported at the message validation level.

File: aries_cloudagent/protocols/out_of_band/v1_0/messages/invitation.py (hunk 1 above)
Comment (shaangill025) [reply aries_cloudagent/protocols/out_of_band/v1_0/messages/invitation.py]: I have created `DIDValidation` pattern [less retrictive] in `messaging/valid.py`.

**Review by andrewwhitehead — APPROVED** [review]

**Pull request merged**